### DISCUSSION TITLE
Request: 任意のページをPDF出力できるようにする

### DISCUSSION THREAD

**Discussion by Akira-Demizu** [growilabs/growi#7731] [Ideas / Requests] [answered]
## Informations

**Is your feature request related to a problem? Please describe.**
GrowiのページをPDFとして外部に共有または提出したい場合がよくあります。
現状は、ブラウザの印刷からPDFにしてダウンロードしております。
しかし、この場合PDFには不要なwebページの情報が記録されております。
また、表示領域も余白が多いため、PDFとして利用できる場面が制限されております。
Growi本来の機能としてはなくても問題ないと思いますが、あるとよりGrowiが利用できる場面が増えそうです。

**Describe the solution you'd like**
PDFダウンロードボタンを設けて、PDF出力できるようになるとありがたいです。
表示はwebブラウザで表示されているものをそのままの形だと理想的です。

また、ページ区切りもユーザー側で簡易に設定できるとありがたいです。
現状は、以下のHTMLタグを設置することで区切りはできていますが、もう少し簡易なコードやタグなどになると嬉しいです。
`<div style="page-break-before:always"></div>`

**Describe alternatives you've considered**
代案としては、.txtなどのテキストファイルの出力も考えられます。
しかし、その場合PDF出力を実装する場合に比べて実装工数も増え、Growiの良さも消えてしまいそうな気がしています。

**Comment by miya** [comment]
現在実装中です。https://github.com/weseek/growi/pull/8388

**Comment by yuki-takei** [comment, marked as answer]
[v7.2.1](https://github.com/growilabs/growi/releases/tag/v7.2.1) にて PDF 出力機能が実装されました。管理者設定で PDF コンバーターを有効化することで、ページのコンテキストメニューからの単ページエクスポートや、一括エクスポート機能（Markdown / PDF 形式）をご利用いただけます。

ご要望をお寄せいただきありがとうございました！
